### ISSUE TITLE
Caldav id

### ISSUE THREAD

**Issue by carduz** [dsd-meetme/backend#8]
Since I remove old timeslots each time I perform the sync do we have overflow with the id?

**Comment by Mihica** [comment]
So with 32bit id and lets say million timeslot changes an hour we will overflow in 2^32/10^6/24 = 178 days. And these syncs will be most of the timeslot changes. I'd make it BIGINT. 

```
On Thursday, December 3, 2015 3:03 AM, claudio cardinale <notifications@github.com> wrote:
```

 Since I remove old timeslots each time I perform the sync do we have overflow with the id?—

**Comment by carduz** [comment]
I think that the best solution is use composite primary key (calendar_id, id) wiht id as bigInt

**Comment by carduz** [comment]
what dop you think?

**Comment by carduz** [comment]
fixed 00eaf5d1b56180b0b02dfcd302554cd016bd485f
